go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/mattermost/mattermost-plugin-api
 go 1.13
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200313113657-e2883bfe5f37
+	github.com/blang/semver v3.5.1+incompatible
+	github.com/mattermost/mattermost-server/v5 v5.20.0
```

Comment:
Can you elaborate on this change? I remember struggling with the demo plugin dependencies here, and `e2883bfe5f37` solved the issue (since it's in the future relative to v5.20.0). Does this still work if you point the demo plugin at your changes?

Reply:
As discussed in channel, this is a weird situation, and I don't know how to fix it. It means anyone using plugin-api has to update their own direct import of mattermost-server to be this version (I haven't tried directly importing a newer version).